Makefile

```diff
@@ -143,6 +147,15 @@ format:
 test:
 	@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/test.sh ./cmd/... ./pkg/... ./test/e2e/binary/...
 
+.PHONY: test-e2e
+test-e2e:
+	@go test -timeout=0 -mod=vendor -ldflags $(LD_FLAGS) ./test/e2e/pod \
+		--v -ginkgo.v -ginkgo.progress \
+		--kubeconfig=${KUBECONFIG} \
+		--access-key-id='$(shell cat $(ACCESS_KEY_ID_FILE))' \
```

Comment:
the testmachinery serves the access-key-id as env var with the name `ACCESS_KEY_ID`